xml/System.ComponentModel/VersionConverter.xml

```diff
@@ -147,14 +147,14 @@
         <Parameter Name="destinationType" Type="System.Type" />
       </Parameters>
       <Docs>
-        <param name="context">To be added.</param>
+        <param name="context">An <see cref="T:System.ComponentModel.ITypeDescriptorContext" /> that provides a format context.</param>
         <param name="culture">The culture that supplies formatting information for the conversion..</param>
         <param name="value">The value to be converted..</param>
-        <param name="destinationType">To be added.</param>
+        <param name="destinationType">The <see cref="T:System.Type" /> to convert the value parameter to.</param>
         <summary>
           <para>Converts the specified value to the specified destination type using the specified context and arguments.</para>
         </summary>
-        <returns>To be added.</returns>
+        <returns>An <see cref="T:System.Object" /> that represents the converted value.</returns>
```

Comment:
From what I've seen in other PRs, we could stick with plain English for some data types:

```suggestion
        <returns>An object that represents the converted value.</returns>
```